validation_tests/dyninst/run.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+cd xmas_tree/build
+./xmas_tree
```

Comment:
Its return code indicates success but there is no output in the log from the run phase. It's not strictly necessary but if there's a quick and easy way to validate that the execution did what it was supposed to that might be worth adding.

Reply:
Sorry for the delay, I must not have seen this. No, the log is mostly for human eyes. The pass/fail is strictly by the return code from the script.